plugins/curation/server/__init__.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright 2016 Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+from girder.api import access
+from girder.api.describe import Description, describeRoute
+from girder.api.rest import Resource, loadmodel
+from girder.constants import AccessType
+from girder.utility import mail_utils
+import datetime
+import posixpath
+
+
+CURATION = 'curation'
+
+ENABLED = 'enabled'
+STATUS = 'status'
+TIMELINE = 'timeline'
+ENABLE_USER_ID = 'enableUserId'
+REQUEST_USER_ID = 'requestUserId'
+REVIEW_USER_ID = 'reviewUserId'
+
+CONSTRUCTION = 'construction'
+REQUESTED = 'requested'
+APPROVED = 'approved'
+
+DEFAULTS = {
+    ENABLED: False,
+    STATUS: CONSTRUCTION,
+}
+
+
+class CuratedFolder(Resource):
+
+    @access.public
+    @loadmodel(model='folder', level=AccessType.READ)
```

Comment:
I think we should require user access to see the curation status, and also allow clients with tokens to access this endpoint.  So change

    @access.public

to

    @access.user(scope=TokenScope.DATA_READ)